config/ui-extensions/apigateways/general

```diff
@@ -0,0 +1,11 @@
+resource:
+  kind: APIGateway
+  group: operator.kyma-project.io
+  version: v1alpha1
+urlPath: apigateways
+category: Kyma
+name: APIGateway
+scope: cluster
+description: >-
+  {{[APIGateway CR](https://github.com/kyma-project/api-gateway/blob/mod-dev/config/samples/operator_v1alpha1_apigateway.yaml)}}
```

Comment:
this is referring `mod-dev` branch, shouldn't we have it `main` instead?

Reply:
we could now provide broken link, but instead I provided valid one and put into test-release followup issue to update this link with other documentation links

Reply:
okay please make sure to add this follow-up task so we do not forget to update it.